client/verta/tests/unit_tests/pipeline/test_registered_pipeline.py

```diff
@@ -0,0 +1,206 @@
+# -*- coding: utf-8 -*-
+"""
+Unit tests for the RegisteredPipeline class
+"""
+
+import pytest
+from hypothesis import given, HealthCheck, settings
+
+from tests.unit_tests.strategies import mock_pipeline_definition
+from verta.pipeline import RegisteredPipeline
+
+
+def test_copy_graph(
+    make_mock_pipeline_graph, make_mock_registered_model_version
+) -> None:
+    """Test that the graph of a RegisteredPipeline can be copied"""
+    graph = make_mock_pipeline_graph()
+    pipeline = RegisteredPipeline(
+        pipeline_graph=graph,
+        registered_model_version=make_mock_registered_model_version(),
+    )
+    copied_graph = pipeline.copy_graph()
+    assert copied_graph.steps == graph.steps  # same steps
+    assert copied_graph != graph  # different objects
```

Comment:
If we'd explicitly like to check for different objects, we should check

```suggestion
    assert copied_graph is not graph  # different objects
```

in case we ever override `PipelineGraph.__eq__()`.